spec/views/hq/sponsors/_sponsorships_inactive_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'hq/sponsors/_sponsorships_inactive.html.haml', type: :view do
+  let(:orphan) {build_stubbed :orphan}
+  let(:sponsorships_inactive) {(1..2).map {build_stubbed :sponsorship, active: false, orphan: orphan}}
```

Comment:
While there is no conflict with validations, I'd still prefer to not reuse the orphan. It only saves creation of some in-memory objects, but makes the test on L16 less robust since only one sponsorship needs to have been rendered for it to pass.